features/issues/components/filters/filters.tsx

```diff
@@ -92,51 +131,81 @@ export function Filters() {
 
   return (
     <Container>
-      <Select
-        placeholder="Status"
-        defaultValue="Status"
-        width="8rem"
-        data-cy="filter-by-status"
-      >
-        <Option value={undefined} handleCallback={handleStatus}>
-          --None--
-        </Option>
-        <Option value="Unresolved" handleCallback={handleStatus}>
-          Unresolved
-        </Option>
-        <Option value="Resolved" handleCallback={handleStatus}>
-          Resolved
-        </Option>
-      </Select>
-
-      <Select
-        placeholder="Level"
-        defaultValue="Level"
-        width="8rem"
-        data-cy="filter-by-level"
+      <Button
+        iconSrc="/icons/select-icon-white.svg"
+        iconOptions={IconOptions.leading}
+        style={{
```

Comment:
I'm curious why you chose to use inline styles instead of CSS via styled-components here.

Reply:
The reason I chose to use inline styles was that the Button component among other UI components is used throughout the app. It didn't make sense to change the button component's functionality only to serve the purpose of this instance. Perhaps there's a better way to extend CSS for reusable UI components that I am not aware of?

Reply:
specifically for styled-components^^